src/views/SystemProfile/components/Team/Edit/index.tsx

```diff
@@ -144,35 +149,41 @@ const EditTeam = ({
 
   return (
     <GridContainer className="margin-bottom-10">
-      <BreadcrumbBar variant="wrap">
-        <Breadcrumb>
-          <BreadcrumbLink asCustom={Link} to="/systems">
-            {t('singleSystem.editTeam.systems')}
-          </BreadcrumbLink>
-        </Breadcrumb>
-        <Breadcrumb>
-          <BreadcrumbLink asCustom={Link} to={`/systems/${cedarSystemId}/team`}>
-            {name}
-          </BreadcrumbLink>
-        </Breadcrumb>
-        {action === 'team-member' ? (
-          <>
-            <Breadcrumb>
-              <BreadcrumbLink
-                asCustom={Link}
-                to={`/systems/${cedarSystemId}/team/edit`}
-              >
-                {t('singleSystem.editTeam.title')}
-              </BreadcrumbLink>
-            </Breadcrumb>
-            <Breadcrumb>
-              {t(`singleSystem.editTeam.form.${actionType}.title`)}
-            </Breadcrumb>
-          </>
-        ) : (
-          <Breadcrumb>{t('singleSystem.editTeam.title')}</Breadcrumb>
-        )}
-      </BreadcrumbBar>
+      {flags.systemWorkspaceTeam && !isWorkspace && (
+        // Don't show crumbs in the workspace context
+        <BreadcrumbBar variant="wrap">
+          <Breadcrumb>
+            <BreadcrumbLink asCustom={Link} to="/systems">
+              {t('singleSystem.editTeam.systems')}
+            </BreadcrumbLink>
+          </Breadcrumb>
+          <Breadcrumb>
+            <BreadcrumbLink
+              asCustom={Link}
+              to={`/systems/${cedarSystemId}/team`}
+            >
+              {name}
+            </BreadcrumbLink>
+          </Breadcrumb>
+          {action === 'team-member' ? (
+            <>
+              <Breadcrumb>
+                <BreadcrumbLink
+                  asCustom={Link}
+                  to={`/systems/${cedarSystemId}/team/edit`}
+                >
+                  {t('singleSystem.editTeam.title')}
+                </BreadcrumbLink>
+              </Breadcrumb>
+              <Breadcrumb>
+                {t(`singleSystem.editTeam.form.${actionType}.title`)}
+              </Breadcrumb>
+            </>
+          ) : (
+            <Breadcrumb>{t('singleSystem.editTeam.title')}</Breadcrumb>
+          )}
+        </BreadcrumbBar>
+      )}
 
       <Message />
```

Comment:
This `Message` needs a top margin added. There isn't any space above it if the breadcrumbs aren't there.

![Screenshot 2024-09-24 at 11 21 43 AM](https://github.com/user-attachments/assets/858b1959-0485-4f3e-ae32-189c32e34ac8)